### DISCUSSION TITLE
Connect to Mysql backend

### DISCUSSION THREAD

**Discussion by sebaFP** [appy-one/acebase#107] [Q&A]
Hi!
Is it possible to connect to a mysql backend? If is possible, can you help me please with qn example?
Thanks!

**Comment by appy-one** [comment]
I have not implemented a MySQL storage adapter yet, but you are welcome to roll your own with a [custom storage adapter](https://github.com/appy-one/acebase#using-a-customstorage-backend)

**Reply by jurri** [reply]
AceBase are a great idea.
But u have to write handler for MySQL, Oracle or Postgre...

@sebaFP here is what u need: https://codeforgeek.com/sync-app-mysql-indexeddb/
